src/server.py

```diff
@@ -208,7 +208,9 @@ def _build_pipeline(self):
         pipeline_volume = self._DEFAULT_VOLUME
         if self.volume is not None:
             pipeline_volume = self.volume
-        pipeline_fade_in = self._DEFAULT_FADE_IN_MS
+        pipeline_fade_in = 0
+        if self.loop:
+            pipeline_fade_in = self._DEFAULT_FADE_IN_MS
```

Comment:
Thank you. Would you mind doing this also for fade-out? :-)

Reply:
Just to point out, a better solution would be to not add the control values if they are no necessary, but this solution could wait.

Reply:
sure!